packages/babylon-ts-sdk/docs/pegin-flow.md

```diff
@@ -0,0 +1,172 @@
+# Peg-in Flow
+
+This guide walks through creating a peg-in transaction step by step.
+
+## Overview
+
+The peg-in flow deposits BTC into a vault. The process involves:
```

Comment:
We should never refer to "vault" just by itself as it might be confused with Ethereum vaults. We should always say "BTC Vault"